### PR TITLE
Fixes for northwestern

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

File: assessmentModel/src/androidLibMain/kotlin/android.kt [hunk 1]
```diff
@@ -32,6 +29,8 @@ actual object UUIDGenerator {
 }
 
 actual object DateGenerator {
-    actual fun nowString(): String = LocalDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)
-    actual fun currentYear(): Int = LocalDate.now().year
+    actual fun nowString(): String {
+        return ZonedDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)
+    }
+    actual fun currentYear(): Int = ZonedDateTime.now().year
```
Comment (syoung-smallwisdom) [inline assessmentModel/src/androidLibMain/kotlin/android.kt:35]: Complete and total nit. Why the different syntax styles? (= vs. {})

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt [hunk 2]
```diff
@@ -70,6 +70,13 @@ open class AssessmentFragment : Fragment() {
                     this.showStep(showNodeState)
                 }
             })
+        start()
+    }
+
+    /**
+     * Starts the Assessment by calling start on the [viewModel]. This is called in onActivityCreated.
+     */
+    open fun start() {
```
Comment (syoung-smallwisdom) [inline presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt:79]: Wondering if this should follow the pattern we used elsewhere of:

```

open fun willStart() {
}

open fun didStart() {
}

// and then above:
willStart()
viewModel.start()
didStart()
```

Or is this intended to allow them to delay the start until they have asynchronously loaded the database?

**Review by syoung-smallwisdom — APPROVED** [review]
Couple of questions, otherwise, looks good.

File: presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt (hunk 2 above)
Comment (nategbrown9) [reply presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt:79]: This is to allow them to delay the start until they have asynchronously initialized their engine.

File: assessmentModel/src/androidLibMain/kotlin/android.kt (hunk 1 above)
Comment (nategbrown9) [reply assessmentModel/src/androidLibMain/kotlin/android.kt:35]: Uhh... good question.

**Pull request merged**